### ISSUE TITLE
Cannot make package with nodejs 16.3

### ISSUE THREAD

**Issue by LesserBabka** [Ericsson/codechecker#3353]
Building codechecker master (8880695, [codechecker_version_info.txt](https://github.com/Ericsson/codechecker/files/6636561/codechecker_version_info.txt)) on arch linux with nodejs 16.3 fails with the following error message (complete [build_log.txt](https://github.com/Ericsson/codechecker/files/6636526/build_log.txt))

```
node: ../src/coroutine.cc:134: void* find_thread_id_key(void*): Assertion `thread_id_key != 0x7777' failed.
make[1]: *** [Makefile:85: /home/joki/git/codechecker/web/server//vue-cli/dist] Error 1
make[1]: Leaving directory '/home/joki/git/codechecker/web'
make: *** [Makefile:46: package] Error 2
```

It seems codechecker is using node-fibers, which is not maintained and does not support nodejs 16.x, [node-fibers homepage](https://github.com/laverdet/node-fibers). This seem like a likely reason.

Building with `BUILD_UI_DIST=NO make package` does work.

**Comment by cit1516-hector** [comment]
I have the same issue as @LesserBabka, on : `macOS 10.14.6 (18G9216)`

My error message is exactly the same (excepted for the current directory), and after looking into it, I agree with @LesserBabka's diagnostic : it seems that node-fibers makes the installation process fail on the `make package` step.

I tried to run npm@15 through [nvm (github project)](https://github.com/nvm-sh/nvm) to enable fibers, to no avail.

I would really like to make CodeChecker run, as it seems that it is superior to plain scan-build thanks to its CTU capabilities.
Thank you !

**Comment by csordasmarton** [comment]
I checked the `package-lock.json` file generated by the new npm version and `fiber` is a dependency of `sass-loader`:
```json
"node_modules/sass-loader": {
  "version": "8.0.2",
  "peerDependencies": {
    "fibers": ">= 3.1.0",
  },
  "peerDependenciesMeta": {
    "fibers": {
      "optional": true
    }
  }
},
```

Checking the release notes of the `sass-loader` package it looks like they solved this problem in the [`11.1.1`](https://github.com/webpack-contrib/sass-loader/releases/tag/v11.1.1) release.

Also from [`11.0.0`](https://github.com/webpack-contrib/sass-loader/releases/tag/v11.0.0) release the minimum supported webpack version is `5` but we are using `4`. So we also have to upgrade this package too.
